index.html

```diff
@@ -0,0 +1,248 @@
+  </head>
+  <body>
+    <nav  class="topNav">
+      <div class="container topNavDiv"
+        >
+          <ul>
+            <li >
+              <a href="#"
+                ><ion-icon name="logo-facebook"></ion-icon
+              ></a>
+            </li>
+            <li >
+              <a href="#"
+                ><ion-icon name="logo-twitter"></ion-icon
+              ></a>
+            </li>
+            <li >
+              <a href="#">English</a>
+            </li>
+            <li >
+              <a href="#">My Page</a>
+            </li>
+            <li >
+              <a href="#">Logout</a>
+            </li></ul
+          ></div>
+    </nav>
+    <header class="container header1">
+      <nav class="nav">
+        <div class="container">
+          <img
+            src="./images/5754_scaling_ai_2022_logo70584.jpg"
+            alt="logoimage"
+            class="navbar-brand"
+            id="logo"
+          />
+          <div class="nav-right">
+            <ul >
+              <li >
+                <a href="./about.html" >About</a>
+              </li>
+              <li >
+                <a href="#">Program</a>
+              </li>
+              <li >
+                <a href="#">Join</a>
+              </li>
+              <li >
+                <a href="#">Sponsor</a>
+              </li>
+              <li >
+                <a href="#">News</a>
+              </li>
+              <li >
+                <a href="#" id="campaign">SAI Campaign</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="home container-fluid pb-5">
+        <nav class="container"
+      ><a href="javascript:openMenu()" id="hamburger-btn"
+        ><ion-icon name="menu-outline" class="icons"></ion-icon></a
+    ></nav>
+    <div class="nav-on-top">
+      <ul>
+        <li class="nav-list-items">
+          <a href="javascript:closeMenu()" id="close-btn"
+            ><ion-icon name="close-outline"></ion-icon
+          ></a>
+        </li>
+        <li class="nav-list-items" onclick="closeMenu()">
+          <a href="./about.html">About</a>
+        </li>
+        <li class="nav-list-items" onclick="closeMenu()">
+          <a href="#">Program</a>
+        </li>
+        <li class="nav-list-items" onclick="closeMenu()">
+          <a href="#">Join</a>
+        </li>
+        <li class="nav-list-items" onclick="closeMenu()">
+          <a href="#">Sponsor</a>
+        </li>
+        <li class="nav-list-items" onclick="closeMenu()">
+          <a href="#">News</a>
+        </li>
+        <li class="nav-list-items" onclick="closeMenu()">
+          <a href="#" id="campaign">SAI Campaign</a>
+        </li>
+      </ul>
+    </div>
+        <div id="hello" class="container mt-5"
+          ><h3>"Hello! Fintech world"</h3>
+          <h1 class="mt-4"
+            >APPLYING AI & MACHINE LEARNING TO FINANCE & TECHNOLOGY 2023</h1
```

Comment:
- [ ] @haliljon great work on your design, however, it would be nice if the text size and the section design matched the Behance design,check the comparison below, 

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/205278073-d1739f59-8350-4b9b-9bd0-57cba2189def.png)  |  ![image](https://user-images.githubusercontent.com/61408860/205278174-c9e95c6d-f8bf-4505-bbb7-ba3695b91253.png)